staging/src/k8s.io/cluster-bootstrap/util/secrets/secrets.go

```diff
@@ -63,6 +63,29 @@ func HasExpired(secret *v1.Secret, currentTime time.Time) bool {
 	return false
 }
 
```

Comment:
can we extract the expired parsing to its own method (e.g. `GetExpiration() (*time.Time, error)`) and have HasExpired() call that instead of duplicating it?

Reply:
of course, updated